### PR TITLE
Port 4616 resync by schedule mode

### PR DESCRIPTION
**Pull request by yairsimantov20**
# Description

What - A way to run the resync periodically regardless to the changes in the port app config
Why - Some of the customers dont want the live events feature due to security issues
How - Used already implemented feature that runs a function every x seconds

## Type of change

- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: port_ocean/ocean.py [hunk 1]
```diff
@@ -49,13 +50,32 @@ def __init__(
...
+        self,
+    ) -> None:
+        async def sync_raw_all() -> None:
+            logger.info("Starting scheduled resync")
+            await self.integration.sync_raw_all()
+            logger.info("Finished scheduled resync")
+
+        interval = self.config.scheduled_resync_interval
+        if interval is not None:
+            logger.info(f"Setting up scheduled resync (interval: {interval} minutes)")
```
Comment (MPTG94) [inline port_ocean/ocean.py:63]:
```suggestion
            logger.info(f"Setting up scheduled resync, the integration will automatically perform a full resync every {interval} minutes)")
```

**Review by MPTG94 — COMMENTED** [review]
Apply fixes and merge

**Review by MPTG94 — APPROVED** [review]
Didn't approve last time

**Review by Tankilevitch — APPROVED** [review]
LGTM

**Pull request merged**